doc/contributing/hw/methodology.md

```diff
@@ -226,7 +226,7 @@ These are discussed in the [Getting Started Designing Hardware](./design.md) doc
 ## FPGA vs Silicon
 
 One output of the OpenTitan project will be silicon instantiations of hardware functionality described in this open source repository.
-The RTL repository defines design functionality at a level satisfactory to prove the hardware and software functionality in an FPGA (see [user guides]({{< relref "doc/ug" >}})).
+The RTL repository defines design functionality at a level satisfactory to prove the hardware and software functionality in an FPGA (see [user guides](https://docs.opentitan.org/doc/guides/getting_started/)).
```

Comment:
user guides -> getting started guide

Reply:
Splitting user guides into different books is essentially a content change though. This shouldn't be blocking to merging this PR, but we shouldn't forget to fix this in the future.